components/operators/openshift-servicemesh/operator/base/subscription.yaml

```diff
@@ -8,4 +8,3 @@ spec:
   name: servicemeshoperator
   source: redhat-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: patch-me-see-overlays-dir
```

Comment:
The startingCSV probably shouldn't have been included from the start.  Probably just a miss when we pulled it over from the gitops catalog.